### PR TITLE
Add kubevault features pages and data

### PR DESCRIPTION
**Pull request by mohin7**
Signed-off-by: Md Mohin Uddin <mohin@appscode.com>

### CODE REVIEW

File: data/products/kubevault/single-features-data/manage-db-users-privileges.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "id": "manage-db-users-privileges",
+  "title": "Manage DB Users Privilegest",
```
Comment (DRoyyy) [inline data/products/kubevault/single-features-data/manage-db-users-privileges.json]: "title": "Manage DB Users Privileges"

File: data/products/kubevault/single-features-data/storage-backend.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    {
+      "id": 2,
+      "title": "Cloud Storage Backend",
+      "description": "Cloud Storage backend is used to persist Vault's data in Cloud Storage. KubeVault Provides support for multiple Storage Backend from Cloud Providers like Amazon Web Services (AWS) S3, Microsoft Azure Storage Container, Google Cloud Storage etc.",
+      "img": "/assets/images/features/illustrations/cloud-storage-backend.jpg"
+    },
+    {
+      "id": 3,
+      "title": "Database Storage Backend",
+      "description": "Database Storage backend is used to persist Vault's data in Database Storage. KubeVault Provides support for multiple Database Storage Backend like  Amazon Web Services (AWS) S3, Microsoft Azure Storage Container, Google Cloud Storage etc.",
```
Comment (DRoyyy) [inline data/products/kubevault/single-features-data/storage-backend.json]: KubeVault Provides support for multiple Database Storage Backend like MySQL, PostgreSQL, etc.

File: data/products/kubevault/single-features-data/database-secret-engine.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "id": "database-secret-engine",
+  "title": "Database Secret Engine",
+  "meta": "KubeVault lets you enable and configure Database Secret Engine. You can create Secret containing the Database credentials and also create RBAC Role and Role Binding.",
```
Comment (DRoyyy) [inline data/products/kubevault/single-features-data/database-secret-engine.json]: and also create RBAC Role and RoleBinding.

File: data/products/kubevault/single-features-data/azure-secret-engine.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
+{
+  "id": "azure-secret-engine",
+  "title": "Azure Secret Engine",
+  "meta": "KubeVault lets you enable and configure Azure Secret Engine. You can create Secret containing the Azure credentials and also create RBAC Role and Role Binding.",
```
Comment (DRoyyy) [inline data/products/kubevault/single-features-data/azure-secret-engine.json]: and also create RBAC Role and RoleBinding.

**Review by DRoyyy — CHANGES_REQUESTED** [review]

**Review by DRoyyy — DISMISSED** [review]
LGTM!

**Review by sakiib — DISMISSED** [review]
LGTM!

**Pull request merged**